src/document-submit.js

```diff
@@ -0,0 +1,111 @@
+/**
```

Comment:
Not entirely sure what should be added to `dep-check-config`, what is a bad inclusion in this case?